documentation/status-logging.md

```diff
@@ -0,0 +1,54 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging though you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
+
+    ```
+    status: {
+        logging-frequency: 2000,
+        status-logging-enabled: true
+    }
```

Comment:
This is net new configuration, right?  This is the first thing that is configurable about the status service?

Might want to have a quick brainstorm about the section name if that's the case.  I haven't thought of anything obviously better than `status`, but, just a good thing to spend a few minutes mulling over.

We could also consider getting rid of the 'enabled' setting and just say that if `logging-frequency` (or `status-log-interval` or something) was set to a non-zero value then that implicitly enables it?  Not 100% sure if that's better, just food for thought.